homeassistant/components/deconz/sensor.py

```diff
@@ -91,76 +87,139 @@ class DeconzSensorDescription(
 
 
 ENTITY_DESCRIPTIONS = {
-    Battery: SensorEntityDescription(
+    AirQuality: [
+        DeconzSensorDescription(
+            key="air_quality",
+            value_fn=lambda device: device.air_quality,  # type: ignore[no-any-return]
+            suffix="",
```

Comment:
You  could also default the suffix to an empty string in the entity description and remove it from all these cases

Reply:
I thought I had figured this out what worked and didn't wrt inheritance and dataclass, tried it and it worked, thanks!